ideal-image-slider.js

```diff
@@ -627,6 +661,10 @@ var IdealImageSlider = (function() {
 		// Preload next images
 		_loadImg(this._attributes.previousSlide);
 		_loadImg(this._attributes.nextSlide);
+
+		//Remove loading class
+		_removeClass(sliderEl, 'iis-loading');
+		_addClass(sliderEl, 'iis-loaded');
```

Comment:
In the future I'll adhere to a vary feature independent PR style. In the name of closing this PR, can we just keep the loading classes in for now?